swim/node.go

```diff
@@ -69,6 +72,9 @@ func defaultOptions() *Options {
 		RollupFlushInterval: 5000 * time.Millisecond,
 		RollupMaxUpdates:    250,
 
+		PartitionHealPeriod:           30 * time.Second,
+		PartitionHealBaseProbabillity: 3,
```

Comment:
Would be useful to provide some documentation on what this value means to the caller, since it's configurable. I understand that the user can go and read the source, but a couple of examples (e.g. with `1`, you get this, with `3` -- this, etcetra) would be a worthy job to document.